src/main/start-main-application/lens-window/application-window/create-first-application-window.injectable.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import createApplicationWindowInjectable from "./create-application-window.injectable";
+
+// Note: Motivation is to create the window with same ID to use stored dimensions
```

Comment:
So it would automatically keep in mind if there's already windows and in case there is, it would use `getRandomId()` and if not, it would default to `first-application-window`?

Interesting.

Reply:
Yes that is sort of what I was suggesting.